demos/inpage/src/WaxInPage.tsx

```diff
@@ -161,6 +173,11 @@ export default class WaxInPage {
 
     const assumedEntryPoint = viewer.connectAssume(EntryPoint__factory, []);
 
+    const assumedAddressRegistry = viewer.connectAssume(
```

Comment:
nitpick - is there a reason `viewer.connectAssume` is used for entrypoint and address registry outside of the `contracts` object? Rather than when creating the object like the contracts. Seems arbitrary but wasn't sure if there was a reason

Reply:
Yeah it's so that we can pass in the address as an argument to the other contracts.